From aecf1e4686bbd50e96bea030bdf7ee4f4882fe5c Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Mon, 2 Oct 2017 17:23:24 -0400
Subject: [PATCH 1/2] WRS:
 0001-grubby-Update-package-versioning-for-TIS-format.patch

Conflicts:
	SPECS/grubby.spec
---
 SPECS/grubby.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SPECS/grubby.spec b/SPECS/grubby.spec
index dd8fa7f..22bff31 100644
--- a/SPECS/grubby.spec
+++ b/SPECS/grubby.spec
@@ -1,6 +1,6 @@
 Name: grubby
 Version: 8.28
-Release: 23%{?dist}
+Release: 23.el7%{?_tis_dist}.%{tis_patch_ver}
 Summary: Command line tool for updating bootloader configs
 Group: System Environment/Base
 License: GPLv2+
-- 
1.9.1

